test/spectests/spectests.cpp

```diff
@@ -293,15 +293,15 @@ class test_runner
                 catch (fizzy::parser_error const& ex)
                 {
                     if (type == "assert_malformed")
-                        pass();
+                        pass(ex.what());
```

Comment:
It would be better to also include the expected error kind, e.g.

```suggestion
                        pass("malformed: " + ex.what());
```

Reply:
Each line already includes the kind of assertion it tests:
```
Line 29: assert_invalid PASSED invalid index of an exported function
```
```
Line 6: assert_malformed PASSED invalid wasm module prefix
```

Reply:
Ah ok. This is not visible in the added CTest test.